website/docs/docs/guides/migration-guide/upgrading-to-0-19-0.md

```diff
@@ -20,16 +20,17 @@ title: "Upgrading to 0.19.0"
 
 Please be aware of the following changes in v0.19.0:
 
-- dbt artifacts have a new schema. From here on, artifact schemas are officially versioned at **schemas.getdbt.com**. Future breaking changes will be limited to minor version releases.
-- Defer, a beta feature introduced in v0.18.0, has subtly changed to better support the "Slim CI" use case.
+1. dbt artifacts have a new schema. From here on, artifact schemas are officially versioned at **schemas.getdbt.com**. Future breaking changes will be limited to minor version releases. Some dbt classes, such as the `Result` object, have associated breaking changes.
```

Comment:
I know this is a PR for `next`, but schemas.getdbt.com doesn't have any content — is that expected for now?

Reply:
yup, expected for now — it'll have something there by the time of final release (: